nncf/common/accuracy_aware_training/training_loop.py

```diff
@@ -435,8 +440,12 @@ def create_accuracy_aware_training_loop(nncf_config: NNCFConfig,
     """
```

Comment:
The create_accuracy_aware_training_loop function has the following signature in the documentation: 
```training_loop = create_accuracy_aware_training_loop(nncf_config, compression_ctrl, original_model_accuracy)```. Could you add `original_model_accuracy` or `uncompressed_model_accuracy` as the optional parameter of this function.